src/http/RequestMethod.cc

```diff
@@ -251,6 +257,11 @@ HttpRequestMethod::respMaybeCacheable() const
         return ??;
 #endif
 
+    // draft-ietf-httpbis-safe-method-w-body
+    case Http::METHOD_QUERY:
+        // TODO: To cache, add request content to our cache key (see Draft section 2.4).
+        return false;
```

Comment:
All other such cases in this function are wrapped in never-true macros like `WHEN_CC_NOCACHE_DOES_REVALIDATES_IS_CONFIRMED`. I suggest that we do the same here for consistency sake, but I do not insist on it (because the associated performance gain, if any, is going to be negligible).